src/read/pe/section.rs

```diff
@@ -324,6 +325,11 @@ impl<'data> SectionTable<'data> {
         self.iter()
             .find_map(|section| section.pe_data_containing(data, va))
     }
+
+    /// Return the section that contains a given virtual address in a PE file.
+    pub fn section_at(&self, va: u32) -> Option<&'data ImageSectionHeader> {
```

Comment:
```suggestion
    /// Return the section that contains a given virtual address.
    pub fn section_containing(&self, va: u32) -> Option<&'data ImageSectionHeader> {
```
Still need to rename this please (sorry, I should have left it as a review comment originally). And no longer need to prefix with `pe_` like I originally requested because this is no longer using the PE file range.

Reply:
OK, I'm applying your suggestion.